pkgs/tools/misc/hyperledger-fabric/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "hyperledger-fabric-${version}";
```

Comment:
```suggestion
  pname = "hyperledger-fabric";
```
`buildGoPackage` can have just a `pname` and a `version` now.

Reply:
I forgot about this, fixed. Thanks!